example.html

```diff
@@ -118,13 +118,11 @@ <h6>Who Gets the Points?</h6>
 
 		// Note that this example is using the key in two places, here and below
 		branch.init('key_live_feebgAAhbH9Tv85H5wLQhpdaefiZv5Dv', function(err, data) {
-			document.getElementsByClassName('info')[0].innerHTML = JSON.stringify(data);
+			// Escape any HTML in the output to avoid XSS.
+			$('.info').text(JSON.stringify(data));
 		});
 	</script>
 	<script type="text/javascript">
-		var info = $('.info');
```

Comment:
These variables were completely unused.

Reply:
Incorrect. The request and response fields were. Changing them to use ids instead of classes though, since they're each unique.